scalableminds/webknossos
# Small Bug fixes

Mailable description of changes (needs to be understandable by webknossos mailing list people):
- Fixes two bugs (weird Bounding Box behavior & volume annotation while moving)

Steps to test:
- Move in volume mode -> No volume annotation
- Enter user Bounding Box (Tracing / Bounding Box) & move around (in Viewport & 3D View) -> BB should not vanish unexpectidly

Issues:
- fixes #1664 & #1690

------
